ban/http/schema.py

```diff
@@ -24,6 +24,15 @@
         'url': 'https://adresse.data.gouv.fr/api-gestion/',
     },
     'paths': {},
+    'tags': [
+        {'name': 'Municipality'},
+        {'name': 'Postcode'},
+        {'name': 'Group'},
+        {'name': 'Housenumber'},
+        {'name': 'Position'},
+        {'name': 'Diff'},
+        {'name': 'User'},
+    ],
```

Comment:
>  They allow to group all routes by model.

That's what I mean. That's a SwaggerUI issue if SwaggerUI issue does not group routes by Resource. And it may have a fix on SwaggerUI side for that…